packages/common/src/locales/en.json

```diff
@@ -22,7 +22,8 @@
 		"INVALID_SIGNATURE": "Invalid signature",
 		"NOT_IMPLEMENTED": "Not implemented",
 		"SITE_KEY_MISSING": "SITE KEY missing",
-		"ACCOUNT_NOT_FOUND": "Account not found"
+		"ACCOUNT_NOT_FOUND": "Account not found",
+		"INVALID_TIMESTAMP": "Invalid timestamp"
```

Comment:
do we have the translations added for the new strings?

Reply:
Yes, they were already present but I needed to move them into the `GENERAL` section which I've now done.

Reply:
I've also added a unit test to catch any missing translation keys

Reply:
https://github.com/prosopo/captcha/blob/5c4ace994c7208e8b25c73473c5d16487a8ef94a/packages/common/src/tests/locales.unit.test.ts

Reply:
This stuff is already in due to the locale PR.